xml/installation.xml

```diff
@@ -16,10 +16,7 @@
  <info>
   <abstract>
    <para>
-    The &hpcm; for &sles; comes with preconfigured system roles. These
-    roles provide a set of preselected packages typical for the specific role,
-    and an installation workflow that configures the system to make
-    the best use of system resources based on a typical use case for the role.
+    This chapter contains basic information about installing and upgrading the &hpcm; for &sles;.
```

Comment:
It is more like:
_... about installing SLES for HPC use cases or upgrading from SLE HPC to SLES._
If our documentation guide lines allow I'd much rather start with a problem statement - along the lines:
_The SLE HPC product has been merged into SLES for ... SP6. This chapter describes how to install SLES for HPC use cases as well as upgrading a SLE HPC system to SLES 15 SP6._
I'm not sure if we really need this intro though - our documentation guidelines may require this: However, the chapter title immediately after the Intro repeats the purpose so why do we have to repeat the same message twice in a row?